app/services/pipeline_report_service.rb

```diff
@@ -236,6 +236,13 @@ def get_pipeline_status(pipeline_run_id)
     ]
   end
 
+  def pipeline_run_succeeded?(pipeline_run)
+    # This condition is carried over from the previous version of the report page (report_helper.rb ln95).
```

Comment:
nit: remove the line... people should be able to track it from the git history. The line will become obsolete with any changes to the other file.

Reply:
Done